deepmd/pd/utils/dataloader.py

```
@@ -0,0 +1,358 @@
+    print_summary,
+    prob_sys_size_ext,
+    process_sys_probs,
+)
+
+log = logging.getLogger(__name__)
+# paddle.multiprocessing.set_sharing_strategy("file_system")
+
+
+def setup_seed(seed):
+    paddle.seed(seed)
+    os.environ["FLAGS_cudnn_deterministic"] = "True"
```

Ruff (0.8.0)

Use capitalized environment variable `FLAGS_CUDNN_DETERMINISTIC` instead of `FLAGS_cudnn_deterministic`

Replace `FLAGS_cudnn_deterministic` with `FLAGS_CUDNN_DETERMINISTIC`

(SIM112)